src/asm/utils/16_read_disk.asm

```diff
@@ -31,15 +31,20 @@ maybe_retry:
 	jnz again
 	jmp disk_error
 done:
+	lea si, DISK_SUCCESS_MESSAGE
+	call print_string_16
 	ret
-
 # Print the return code of interrupt
 disk_error:
-	lea bx, DISK_ERROR_MESSAGE
-	call print_string
-disk_error_loop:
-	jmp disk_error_loop
+	lea si, DISK_ERROR_MESSAGE
+	call print_string_16
+	xor dx, dx
+	mov dl, ah
+	# See http://www.ctyme.com/intr/rb-0606.htm#Table234 for return code meaning
+	call print_hex_16
+	hlt
 
-DISK_ERROR_MESSAGE:	.asciz "Disk read message! Return Code: "
+DISK_ERROR_MESSAGE:		.asciz "Disk read message! Return Code: "
```

Comment:
Add \r\n

Reply:
done

Reply:
Also, change it to something that indicates that it's an error, like "Disk read error"

Reply:
done